pages/flashbar/interactive.page.tsx

```diff
@@ -67,7 +85,11 @@ export default function InteractiveFlashbar() {
           </Button>
           <Button onClick={() => removeAndAddToBottom('error')}>Add And Remove</Button>
         </SpaceBetween>
-        <Flashbar items={items} {...restProps} />
+        <ScreenshotArea>
```

Comment:
Best practice is to separate functional and screenshot testing pages

Future people will appreciate that adding an extra example to functional tests will not require manual approval in pipeline, because a screenshot test got affected

Reply:
Done, thanks for the suggestion. I added `collapsible.visual-tests.page.tsx`.